apps/site/src/pages/account/api-keys-panel/api-keys-list/api-key-table-row.tsx

```diff
@@ -0,0 +1,116 @@
+import { Box, TableCell, TableRow, Typography } from "@mui/material";
+import { useMemo } from "react";
+
+import { CODE_FONT_FAMILY } from "../../../../theme/typography";
+import { useApiKeys } from "../api-keys-context";
+import { ApiKeyItemProps } from "../types";
+import { RowActions } from "./api-key-table-row/row-actions";
+import { formatDateForDisplay } from "./shared/format-date-for-display";
+import { NewIndicator } from "./shared/new-indicator";
+import { NewlyCreatedApiKeyCard } from "./shared/newly-created-api-key-card";
+
+const MaskedPublicId = ({ publicId }: { publicId: string }) => {
+  return (
+    <Typography
+      sx={{
+        fontFamily: CODE_FONT_FAMILY,
+        color: "gray.50",
+        fontSize: 14,
+        lineBreak: "anywhere",
+      }}
+    >
+      b10ck5.
+      <Box component="span" sx={{ color: "gray.90" }}>
+        {publicId}
+      </Box>
+      .xxxxx
+    </Typography>
+  );
+};
+
+const DateText = ({ date }: { date?: Date | null }) => {
+  const formattedDates = formatDateForDisplay(date);
+
+  return (
+    <Typography>
+      {formattedDates.relative}
+      <Typography sx={{ color: "gray.70", fontSize: 14 }}>
+        {formattedDates.exact}
+      </Typography>
+    </Typography>
+  );
+};
+
+export const ApiKeyTableRow = ({
+  apiKey: { displayName, publicId, createdAt, lastUsedAt },
+  matchingNewlyCreatedKey,
```

Comment:
I think using it this way makes more sense, because `apiKey` meant to be the `UserFacingApiKeyProperties` data. But renamed it as `fullKeyValue` here, so it's more self-explanatory https://github.com/blockprotocol/blockprotocol/pull/1257/commits/c173b818aae79e1a1b17a92d3d5f56f1fa392550

Reply:
Let's chat about this offline as not sure I understand. Don't see how fullKeyValue makes sense here either.

Reply:
I see now – fullKeyValue is the actual apiKey. This is okay.